KioskApplication/src/main/java/controller/SearchController.java

```diff
@@ -0,0 +1,92 @@
+package controller;
+
+import com.jfoenix.controls.JFXComboBox;
+import com.jfoenix.controls.JFXTextField;
+import database.objects.Node;
+import entity.MapEntity;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import javafx.collections.transformation.SortedList;
+import javafx.event.EventHandler;
+import javafx.fxml.FXML;
+import javafx.scene.control.TableView;
+import javafx.scene.control.Tooltip;
+import javafx.scene.input.KeyEvent;
+
+import java.util.function.Predicate;
+
+public class SearchController {
+
+    @FXML
+    private JFXComboBox<Node> cbNodes;
```

Comment:
This combo box should show the Long Name for the nodes.

Reply:
this is certainly up to changes